database/migrations/2014_10_12_000001_modify_users_table.php

```diff
@@ -11,7 +11,7 @@ class ModifyUsersTable extends Migration
     public function up()
     {
         Schema::table('users', function (Blueprint $table) {
-            $table->string('name_last')->after('creator_id');
+            $table->string('name_last')->default('')->after('creator_id');
```

Comment:
I would prefer to not allow this into the migration. Could we update the user factory to include the last name?

Reply:
Its a migration issue, not a factory issue.

Reply:
I would like to prevent blank last names from getting into the database

Reply:
You can combine this migration with the create migration and eliminate the default, the problem is w/`ALTER` specifically and sqlite.

Reply:
That makes sense. The problem with changing the default migration is that it is included with every Laravel installation now. So, I think we need to leave it like it is and then update it.

Reply:
@drewroberts - made a change that limits setting of the default to the `testing` connection only.